src/components/structures/TabbedView.tsx

```diff
@@ -129,56 +119,54 @@ export default class TabbedView<T extends string> extends React.Component<IProps
                 </span>
             </RovingAccessibleButton>
         );
-    }
+    };
 
-    private getTabId(tab: Tab<T>): string {
+    const getTabId = (tab: Tab<T>): string => {
         return `mx_tabpanel_${tab.id}`;
-    }
+    };
 
-    private renderTabPanel(tab: Tab<T>): React.ReactNode {
-        const id = this.getTabId(tab);
+    const renderTabPanel = (tab: Tab<T>): React.ReactNode => {
```

Comment:
I'm more in favour to create a dedicated small components instead of using `renderX` function.